test/lib/mayaUsd/fileio/testSchemaApiAdaptor.py

```diff
@@ -17,13 +17,20 @@
 
 import mayaUsd.lib as mayaUsdLib
 
-from pxr import Tf, Gf, UsdMaya, Usd
+from pxr import Tf, Gf, Usd
 
 from maya import cmds
 import maya.api.OpenMaya as om
 import maya.api.OpenMayaAnim as oma
 from maya import standalone
 
+HAS_USDMAYA = False
+try:
+    from pxr import UsdMaya
```

Comment:
This python module is not present if you don't build the Pixar plugin. It is used by **one** of the tests in this file (which exists in the maya-usd core). So skip that test if we don't have UsdMaya.

Reply:
Why the issue came up with AR2.0? I can’t find the link?